### ISSUE TITLE
C# windows tests failing on master due to unreliable python3.8 installation

### ISSUE THREAD

**Issue by jtattermusch** [grpc/grpc#21338]
Recently I've been seeing very high flakiness rate for C# build on master.

I suspect this is caused by https://github.com/grpc/grpc/pull/21271, which adds python 3.8 installation to the prepare script.
The python installation sometimes causes the prevents golang from being installed (which in turn fails the C# build).

Recent logs
https://source.cloud.google.com/results/invocations/480015d0-1f5c-4340-9aab-13e2d5c13582/log
https://source.cloud.google.com/results/invocations/0458ce79-7075-4245-96ac-36fa839b57bd/log
https://source.cloud.google.com/results/invocations/ad9bfda0-7053-462f-8c8a-7f5292e82b0e/log

```
WARNING: Another installation currently in progress. Try again later.
ERROR: Running ["C:\Windows\System32\msiexec.exe" /i "T:\tmp\chocolatey\golang\1.13.1\go1.13.1.windows-amd64.msi" /qn /norestart ] was not successful. Exit code was '1618'. Exit code indicates the following: Another installation currently in progress. Try again later..
  golang may be able to be automatically uninstalled.
The install of golang was NOT successful.
Error while running 'C:\ProgramData\chocolatey\lib\golang\tools\chocolateyinstall.ps1'.
 See log for details.

Chocolatey installed 0/1 packages. 1 packages failed.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).

Failures
 - golang (exited 1618) - Error while running 'C:\ProgramData\chocolatey\lib\golang\tools\chocolateyinstall.ps1'.
 See log for details.
```

**Comment by jtattermusch** [comment]
Looking at build history the C# test jobs really started to be flaky once #21271 was merged.

**Comment by jtattermusch** [comment]
This has been resolved (by merging #21339).
